app/Repositories/Eloquents/ReviewBookEloquentRepository.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Repositories\Eloquents;
+
+use App\Eloquent\Review;
+use App\Repositories\Contracts\ReviewBookRepository;
+use Auth;
+
+class ReviewBookEloquentRepository extends AbstractEloquentRepository implements ReviewBookRepository
+{
+    public function model()
+    {
+        return new Review;
+    }
+
+    public function store($data = [])
+    {
+        $data['user_id'] = Auth::user()->id;
+
+        return $this->model()->create($data);
+    }
+
+    public function show($data = []){
+        $review = Review::where('book_id', '=', $data)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/ReviewBookEloquentRepository.php</br>Opening brace should be on a new line